grr/server/grr_response_server/fleetspeak_utils.py

```diff
@@ -16,6 +16,7 @@
 from fleetspeak.src.common.proto.fleetspeak import common_pb2 as fs_common_pb2
 from fleetspeak.src.common.proto.fleetspeak import system_pb2 as fs_system_pb2
 from fleetspeak.src.server.proto.fleetspeak_server import admin_pb2
+from fleetspeak.src.server.proto.fleetspeak_server.resource_pb2 import ClientResourceUsageRecord
```

Comment:
Per pystyle, do not import individual members from modules, instead import the whole module `resource_pb2` and reference the member `resource_pb2.ClientResourceUsageRecord`.